src/Bridge/Doctrine/MongoDbOdm/Filter/BooleanFilter.php

```diff
@@ -36,7 +36,7 @@ final class BooleanFilter extends AbstractFilter
 {
     use BooleanFilterTrait;
 
-    const DOCTRINE_BOOLEAN_TYPES = [
+    public const DOCTRINE_BOOLEAN_TYPES = [
```

Comment:
I'm -1 on this `public const` thing it's just noise IMO. Symfony is not using it for now?

Reply:
We should now pay attention to the visibility of our constants and define them explicitly. It's by default in the PHP-CS-Fixer ruleset `@PHP71Migration`.

Reply:
> It's by default in the PHP-CS-Fixer ruleset @PHP71Migration.

But we are already using both `@PHP71Migration` and `@PHP71Migration:risky` rulesets?

Reply:
```
visibility_required [@PSR2, @Symfony, @PhpCsFixer, @PHP71Migration]

Visibility MUST be declared on all properties and methods; abstract and final MUST be declared before the visibility; static MUST be declared after the visibility.

Configuration options:

- elements (a subset of ['property', 'method', 'const']): the structural elements to fix (PHP >= 7.1 required for const); defaults to ['property', 'method']
```

So it's not enabled by default. If we're going to make this change, we must change that option.

Reply:
By default constants are public and it won't change, we should defined them if they're not public IMO.

https://secure.php.net/manual/en/migration71.new-features.php#migration71.new-features.class-constant-visibility

Reply:
Yep but because of the alphabetical order in the configuration, the `@Symfony` ruleset has the advantage and it uses the default configuration: `method` + `property` vs. `method` + `property` + `const` for the `@ PHP71Migration` ruleset.

Reply:
> By default constants are public and it won't change

By default method are public but you define a visibility explicitly even if it is public.

Reply:
Okay, let's add the correct configuration for this rule then.

Reply:
> Yep but because of the alphabetical order in our configuration, the @Symfony ruleset has the advantage

That is indeed unexpected, but might as well since I don't think we would ever find a "correct" order. :laughing:

Reply:
I just updated the `.php_cs.dist` file.